### ISSUE TITLE
Module missing from old Veil-Evasion

### ISSUE THREAD

**Issue by ghost** [Veil-Framework/Veil#246]
hello guys last days i downloaded veil evasion to use the module python shellcode download inject but then i noticed that there is new version (Veil 3.1) which contains both Veil Evasion and Veil Ordnance so i did download the new version and installed it and everything worked perfect but when i looked for the module i wanted to usei didn't find it
i was wondering why you removed this
link : 
https://github.com/Veil-Framework/Veil-Evasion/blob/master/modules/payloads/python/shellcode_inject/download_inject.py

and also when i tried using veil evasion again (i reinstalled it using the setup) it fails when creating the output file but don't mind it 

**In conclusion: just consider adding the module if possible 
if not i'll create a new issue showing the problem i got using the old veil evasion**

**Comment by ChrisTruncer** [comment]
I am always open to pull requests, I haven't had the time to add it back in yet, but if you want to submit a pull request with it, I'd be happy to review and merge it in.
